docs/default_disaster.js

```diff
@@ -0,0 +1,14 @@
+export {
+  DEFAULT_DISASTER
```

Comment:
Can we one-line this?

Also, could violate convention and put at end of file so line user has to edit is closer to top?

Reply:
I thought about that too. Went for it!